modules/run-consul/run-consul

```diff
@@ -287,6 +290,7 @@ autostart=true
 autorestart=true
 stopsignal=INT
 user=$consul_user
+environment=${environment[@]}
```

Comment:
Does this work OK if it's empty?

Reply:
Yes. Also I fixed an issue where multiple values are not joined with space.